### DISCUSSION TITLE
Welcome to rdl-standard Discussions!

### DISCUSSION THREAD

**Discussion by ldodds** [GFDRR/rdl-standard#1] [General]
## 👋 Welcome!

We’re testing the use of Github Discussions as a means to connect more broadly with members of our community. 

We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together.

Feel free to use this forum for general discussion about the Risk Data Library Standard

**Comment by matamadio** [comment]
@pzwsk @stufraser1 we should remember to use this "Discussion" space as the most fit to engage with partners and community, instead of issue board.
